### PR TITLE
Add live555 recipe, gcc2 fixes included

### PR DESCRIPTION
**Pull request by Barrett17**

### CODE REVIEW

File: media-video/live555/live555-2016.01.29.recipe [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+REVISION="1"
+SOURCE_URI="git+https://github.com/xanview/live555#d9e97d7953d531a243a8372870bdb6c7a9bb80cb"
+PATCHES="live555-2016.01.29.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	live555$secondaryArchSuffix = $portVersion
+	lib:live555 = $portVersion
```
Comment (pulkomandy) [inline media-video/live555/live555-2016.01.29.recipe]: missing $secondaryArchSuffix

File: media-video/live555/live555-2016.01.29.recipe [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:ld$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./genMakefiles haiku
+	make -j4
```
Comment (pulkomandy) [inline media-video/live555/live555-2016.01.29.recipe]: please use $jobArgs instead of hardcoding -j4

**Comment by korli** [comment]
Please use an archive for SOURCE_URI, and provide a checksum.
The patch doesn't indicate gcc2 fixes like your PR seems to imply.

File: media-video/live555/live555-2016.01.29.recipe (hunk 2 above)
Comment (korli) [inline media-video/live555/live555-2016.01.29.recipe]: HOMEPAGE should include a protocol.

**Comment by korli** [comment]
Forget my remark about gcc2 fixes, I didn't notice the second patch.
Please add x86_gcc2 as secondary arch.

**Comment by Barrett17** [comment]
I think it can now be merged.

**Pull request merged**